content/content_service_test.go

```diff
@@ -554,163 +572,168 @@ func cleanDB(db neoutils.CypherRunner, t *testing.T, assert *assert.Assertions)
 		audioContentUUID,
 	}
 
-	qs := []*neoism.CypherQuery{}
+	qs := []*cmneo4j.Query{}
 	for _, uuid := range uuids {
-		qs = append(qs, &neoism.CypherQuery{
-			Statement: `MATCH (t:Thing {uuid:{uuid}}) DETACH DELETE t`,
-			Parameters: neoism.Props{
+		qs = append(qs, &cmneo4j.Query{
+			Cypher: `MATCH (t:Thing {uuid: $uuid}) DETACH DELETE t`,
+			Params: map[string]interface{}{
 				"uuid": uuid,
 			},
 		})
 	}
 
-	err := db.CypherBatch(qs)
+	err := d.Write(qs...)
 	assert.NoError(err)
 }
 
-func writeContentPackageContainsRelation(db neoutils.NeoConnection, cpUUID string, UUID string, t *testing.T, assert *assert.Assertions) {
-	var writeRelation string
-	var qs []*neoism.CypherQuery
-
-	writeRelation = `
-	MATCH (cp:ContentPackage {uuid:{cpUUID}}), (t {uuid:{UUID}})
+func writeContentPackageContainsRelation(d *cmneo4j.Driver, cpUUID string, UUID string, assert *assert.Assertions) {
+	writeRelation := `
+	MATCH (cp:ContentPackage {uuid: $cpUUID}), (t {uuid:$UUID})
 	CREATE (cp)-[pred:CONTAINS]->(t)
 	`
 
-	qs = []*neoism.CypherQuery{
+	qs := []*cmneo4j.Query{
 		{
-			Statement:  writeRelation,
-			Parameters: neoism.Props{"cpUUID": cpUUID, "UUID": UUID},
+			Cypher: writeRelation,
+			Params: map[string]interface{}{
+				"cpUUID": cpUUID,
+				"UUID":   UUID,
+			},
 		},
 	}
 
-	err := db.CypherBatch(qs)
+	err := d.Write(qs...)
 	assert.NoError(err)
 }
 
-func writeNodeWithLabels(db neoutils.NeoConnection, UUID string, labels string, t *testing.T, assert *assert.Assertions) {
-	var writeThingWithLabelsQuery string
-	var qs []*neoism.CypherQuery
+func writeNodeWithLabels(d *cmneo4j.Driver, UUID string, labels string, assert *assert.Assertions) {
+	writeThingWithLabelsQuery := `CREATE (n:` + labels + `{uuid: $uuid})`
 
-	writeThingWithLabelsQuery = `CREATE (n:` + labels + `{uuid: {uuid}})`
-
-	qs = []*neoism.CypherQuery{
+	qs := []*cmneo4j.Query{
 		{
-			Statement:  writeThingWithLabelsQuery,
-			Parameters: neoism.Props{"uuid": UUID},
+			Cypher: writeThingWithLabelsQuery,
+			Params: map[string]interface{}{
+				"uuid": UUID,
+			},
 		},
 	}
 
-	err := db.CypherBatch(qs)
+	err := d.Write(qs...)
 	assert.NoError(err)
 }
 
-func writeRelationship(db neoutils.NeoConnection, contentID string, conceptID string, t *testing.T, assert *assert.Assertions) {
-	var annotateQuery string
-	var qs []*neoism.CypherQuery
-
-	annotateQuery = `
-		MERGE (content:Thing{uuid:{contentId}})
+func writeRelationship(d *cmneo4j.Driver, contentID string, conceptID string, assert *assert.Assertions) {
+	annotateQuery := `
+		MERGE (content:Thing{uuid:$contentId})
 		MERGE (concept:Thing) ON CREATE SET concept.uuid = {conceptId}
```

Comment:
This one should be updated as well.

> ```suggestion

		MERGE (concept:Thing) ON CREATE SET concept.uuid = $conceptId
```